youtube/generic.md

```diff
@@ -7,9 +7,9 @@ Don’t forget to support my channel by subscribing it! It’s free: https://www
 I have another youtube channel for short clips, please **take a moment subscribing it: https://www.youtube.com/eddiejaoudetv**
 
 ## Join Our Community
-Join our inclusive open source community EddieHub now for **free** at http://eddiehub.org
+Join our inclusive open source community EddieHub now for **free** at https://eddiehub.org
```

Comment:
@krishdevdb the above link too cintains *www*

Reply:
> @krishdevdb the above link too cintains _www_

@krishdevdb better change every link https://eddiehub.org in the doc to https://www.eddiehub.org/